### PR TITLE
Problem: priority test result affect by base fee dropping

### PR DESCRIPTION
**Pull request by yihuang**
## Description

Solution:
- increase the scale of priority numbers to make it more stable.

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by danburck — APPROVED** [review]
LGTM @yihuang, can you fix the linter?

File: tests/integration_tests/test_priority.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def tx_priority(tx, base_fee):
 def test_priority(ethermint: Ethermint):
     """
     test priorities of different tx types
+
+    use relatively large priority number to counter the effect of base fee change during the testing.
```
Comment (yihuang) [inline tests/integration_tests/test_priority.py:34]:
```suggestion
    use a relatively large priority number to counter
    the effect of base fee change during the testing.
```

**Comment by yihuang** [comment]
> LGTM @yihuang, can you fix the linter?

done

**Review by fedekunze — APPROVED** [review]
utACK

**Pull request merged**